docs/section4/4-11.md

```diff
@@ -0,0 +1,67 @@
+    - Reset hvac type check (set to true below if there is a system type 5,6,7, or 8 on the floor) boolean variable to FALSE: `hvac_type_check = FALSE`
+    - Get the floor name associated with the zone: `floor_name = match_data_element(B_RMR,Zone,zone_p.id).floor_name`
+    - Get dictionary of zones and hvac systems associated with the floor name: `dict_of_zones_hvac_systems_serving_specific_floor = get_dict_of_zones_hvac_sys_serving_specific_floor(floor_name,B_RMR)`
+    - Create list of hvac system serving the floor from the dictionary: `list_hvac_sys_serving_floor = list(dict_of_zones_hvac_systems_serving_specific_floor.values())`
+    - Loop through and check if any systems are of type 5, 5b, 6, 6b, 7, 7a, 7b, 7c, 8, 8a, 8b, 8c: `for hvac_flr in list_hvac_sys_serving_floor:`
+        - Check if the system is of type 5, 5b, 6, 6b, 7, 7a, 7b, 7c, 8, 8a, 8b, 8c: `if baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_flr.id)] in ["SYS-5","SYS-5b","SYS-6, SYS-6b", SYS-7",SYS-7a",SYS-7b",SYS-7c",SYS-8",SYS-8a",SYS-8b",SYS-8c"]:`
+            - Set hvac_type_check boolean variable to TRUE: `hvac_type_check = TRUE`  
+                                
+    **Rule Assertion:** 
+    - Case 1: For each zone, hvac fan schedules match between the baseline and proposed and either single zone systems or multizone systems are both modeled in the baseline and proposed then PASS: `if schedule_mismatch == FALSE and system_type_match_baseline_proposed == TRUE: outcome == "PASS"`
+    - Case 2: For each applicable zone, elif there is a fan schedule mismatch but Section G3.1.1(c) appears to apply then undetermined: `elif schedule_mismatch == TRUE AND baseline_served_by_multizone == FALSE AND hvac_type_check == TRUE AND proposed_served_by_multizone == TRUE: outcome == "UNDETERMINED" and raise_message "THERE IS A FAN OPERATING SCHEDULE MISMATCH BETWEEN THE BASELINE AND PROPOSED BUT SECTION G3.1.1(c) APPEARS APPLICABLE. VERIFY MISMATCH IS APPROPRIATE PER SECTION G3.1.1(c) AND THAT THE FAN OPERATING SCHEDULE IN THE BASELINE IS IN ALIGNMENT WITH THE OCCUPANCY SCHEDULES."`
+    - Case 4: Else if, there is a schedule mismatch between the baseline and proposed then undetermined: `elif schedule_mismatch == TRUE: outcome == "UNDETERMINED" and raise_message "THERE IS A FAN SCHEDULE MISMATCH BETWEEN THE BASELINE AND PROPOSED RMRs FOR THE HVAC SYSTEM(S) SERVING THIS ZONE. FAIL UNLESS TABLE G3.1 SECTION 4 BASELINE COLUMN EXCEPTIONS #1, #2 or #3 IS APPLICABLE"` 
+    - Case 3: Else (schedules match between the baseline and proposed), undertermined: `Else: outcome == "UNDETERMINED" and raise_message "FAN SCHEDULES MATCH BETWEEN THE BASELINE AND PROPOSED RMRs FOR THE HVAC SYSTEM(S) SERVING THIS ZONE. VERIFY THAT MATCHING SCHEDULES ARE APPROPRIATE IN THAT NONE OF THE SECTION 4 BASELINE COLUMN EXCEPTIONS #1, #2 or #3 ARE APPLICABLE"`  
+
```

Comment:
This RDS looks very good to me.
However, I still recommend just checking for case 1 and for all all other scenarios, providing an undetermined outcome with a generic message which could identify the presence of any of these 3 exceptions.
This is adding a lot of checking and work for the RCT and i don't see value of doing all of this because we still get an undetermined outcome for each scenario.

Reply:
No problem. Do we all agree then that we do not need lines 53 to 59? These lines are only to see if exception #3 likely applies.

Reply:
@claperle - my apologies. we had discussed this at the last RDS  call and I forgot about that conversation. I think it is ok to leave lines 53-59, most of the work is being done before and this is simply grouping zones by floor name and then HVAC systems by floor. This looks good as is and I'll approve it